toolbox

```diff
@@ -2264,6 +2268,8 @@ fi
 
 echo "$base_toolbox_command: TOOLBOX_PATH is $TOOLBOX_PATH" >&3
 
+echo "THIS IMPLEMENTATION OF TOOLBOX IS NO LONGER MAINTAINED."
+
```

Comment:
Would be good to also include the advice to use the Go implementation as in the above comment, and a trailing newline character to make the message stand out from the other messages.

Reply:
Good point!